overcooked_ai_js/package.json

```diff
@@ -21,7 +21,7 @@
     "babel-core": "^6.26.3",
     "babel-preset-env": "^1.7.0",
     "babelify": "^8.0.0",
-    "canvas": "^2.5.0",
+    "canvas": "^2.6.0",
     "jest": "^23.6.0",
     "jest-canvas-mock": "^2.1.0",
     "jest-expect-message": "^1.0.2",
```

Comment:
We do plan on completely depricating all of the code in `overcooked_ai_js/`, so your updates shouldn't touch any files in this sub-directory

Reply:
change reversed